app/assets/javascripts/govuk_publishing_components/vendor/lux/lux-reporter.js

```diff
@@ -468,7 +466,7 @@
       maxBeaconUrlLength: getProperty(obj, "maxBeaconUrlLength", 8190),
       maxBeaconUTEntries: getProperty(obj, "maxBeaconUTEntries", 20),
       maxErrors: getProperty(obj, "maxErrors", 5),
-      maxMeasureTime: getProperty(obj, "maxMeasureTime", 60000),
+      maxMeasureTime: getProperty(obj, "maxMeasureTime", 60_000),
```

Comment:
This is interesting... haven't seen a variable of the form `60_000` before, but it's in the original code.